PokemonRampUp/Model/Ability.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Ability.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-02-14.
+//
+
+import Foundation
+
+struct Ability: Decodable {
+    let ability: Species
```

Comment:
Why is the type Species?

Reply:
https://github.com/arriyam/PokemonRampUp/pull/3#discussion_r1106178428